pypasta/Repository/Mbox.py

```diff
@@ -316,8 +316,10 @@ def update(self):
             else:
                 log.error('Mail processor failed!')
 
-    def get_patchwork_ids(self, message_id):
-        return [cached[3] for cached in self.index[message_id]]
+    def get_patchwork_id(self, message_id):
```

Comment:
Please don't introduce code and entirely remove it in the next patch. It's hard for me to review what your code does, when the semantics of the routines changes every other patch.